packages/ses/package.json

```diff
@@ -20,7 +20,7 @@
   },
   "type": "module",
   "main": "./dist/ses.cjs",
-  "module": "./index.js",
+  "module": "./dist/ses.cjs",
```

Comment:
This cannot be `.cjs` or it isn't a module, must either be `ses.umd.js` or `ses.esm.js`.

Reply:
I’ve now tried `ses.umd.js` and it fails with the same effect.

Reply:
Okay, you tried. :smile:

You have my blessing to close this PR.